parameter-api/Dockerfile

```
@@ -0,0 +1,62 @@
+ARG ARCH=armv7hf
+ARG SDK_VERSION=1.1
+ARG REPO=axisecp
+ARG RUNTIME_IMAGE=arm32v7/ubuntu:20.04
+
+FROM $REPO/acap-computer-vision-sdk:$SDK_VERSION-$ARCH as cv-sdk-runtime
+FROM $REPO/acap-computer-vision-sdk:$SDK_VERSION-$ARCH-devel as cv-sdk-devel
```

I haven't tested this, but I think it should work with the updated FROM on line 54.
If it does work and you add this change, make sure to remove RUNTIME_IMAGE in README too.

```suggestion
ARG UBUNTU_IMAGE_VER=20.04

FROM arm32v7/ubuntu:${UBUNTU_IMAGE_VER} as runtime-image-armv7hf
FROM arm64v8/ubuntu:${UBUNTU_IMAGE_VER} as runtime-image-aarch64
FROM $REPO/acap-computer-vision-sdk:$SDK_VERSION-$ARCH as cv-sdk-runtime
FROM $REPO/acap-computer-vision-sdk:$SDK_VERSION-$ARCH-devel as cv-sdk-devel
```